### PR TITLE
MS-1459 task: add package.json

### PR DESCRIPTION
**Pull request by jjrobothamho**
- create initial package.json file

- create .gitignore file to ignore local settings and node modules

### CODE REVIEW

**Comment by jjrobothamho** [comment]
Let me know if these PR's are too granular

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+public
+apps/*/translations/*
+!apps/*/translations/src
+node_modules/
+.env
+.vscode/*
```
Comment (zuzak) [inline .gitignore]: This is fine, but ideally a repo's `.gitignore` should only contain files generated by the things in that repo -- so `node_modules` should go here, but external things that only some developers would come across, like `.idea` and `.vscode`, would be better placed in a [global gitignore](https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) instead.

Comment (jjrobothamho) [reply .gitignore]: Good idea, thanks Douglas. I'll adjust the PR

Comment (sulthan-ahmed) [reply .gitignore]: Useful to know, good stuff Douglas

**Pull request merged**